core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -90,6 +90,7 @@ oppia.directive('audioTranslationBar', [
           $scope.showRecorderWarning = false;
           $scope.audioLoadingIndicatorIsShown = false;
           $scope.checkingMicrophonePermission = false;
+          $scope.showAudioTimer = true;
```

Comment:
change to isAudioTimerShown. Variable names starting with a verb typically denote function names (as it is supposed to do something).

Reply:
Ah -- @nithusha21, "is" is actually a verb, too. Let's go with audioTimerIsShown or similar.